patterns/1-initial/transparent-cross-team-decision-making-using-rfcs.md

```diff
@@ -0,0 +1,101 @@
+## Title
+
+Transparent Cross-Team Decision Making using RFCs
+
+## Patlet
+
+InnerSource projects that want to achieve high participation rates and make the best possible decisions for everybody involved need to find ways to create participatory systems throughout the full software lifecycle. Publishing internal Requests for Comments (RFCs) documents allows for discussions early on in the design process, and increases the chances to build solutions with a high degree of commitment from all involved parties.
+
+## Problem
+
+For an InnerSource project to be healthy, it needs a substantial amount of contributors. These contributors (or teams) maybe have different requirements for the given project. e.g. they may want to add features to the project that are not compatible with one another or lead to an unhealthy bloat in the architecture.
+
+Discovering such disagreements or misunderstanding late in the process, e.g. once the software has already been built, is very costly. These disagreements may lead to frustrations by all parties involved, and can even be disruptive to the health of the collaboration culture in the project. A common situation where such a disagreement surfaces is a change request (pull request) which is open for a very long time because the author of the change request and the maintainers of the project essentially don’t agree that the proposed change should be made at all.
+
+For an InnerSource project this situation happens more frequently when the project is maintained by multiple teams in the company i.e. shared ownership.
+
+## Story (optional)
+
+**TBD**
+
+## Context
+
+- shared ownership by many teams of an InnerSource project
```

Comment:
Another reason could be cross team dependencies where making changes openly can help resolve competing customer/user requirements by making those requirements visible to everyone.